### PR TITLE
Test/service info

### PR DESCRIPTION
**Pull request by msingermann**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -138,6 +138,12 @@
             <artifactId>logback-jackson</artifactId>
             <version>${logback.contrib.version}</version>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/commons-validator/commons-validator -->
+        <dependency>
+            <groupId>commons-validator</groupId>
+            <artifactId>commons-validator</artifactId>
+            <version>1.6</version>
```
Comment (ldebello) [inline pom.xml]: could we move the version to a property?

File: src/main/java/com/mulesoft/amc/supportxapi/exceptions/ServiceNotConfigredException.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package com.mulesoft.amc.supportxapi.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+/**
+ * Exception that is thrown when the service is not found in the services info map.
+ */
+@ResponseStatus(value = HttpStatus.NOT_FOUND)
+public class ServiceNotConfigredException extends RuntimeException {
```
Comment (ldebello) [inline src/main/java/com/mulesoft/amc/supportxapi/exceptions/ServiceNotConfigredException.java]: I think typo error in "ServiceNotConfigredException"

Comment (msingermann) [reply src/main/java/com/mulesoft/amc/supportxapi/exceptions/ServiceNotConfigredException.java]: fixed!

File: pom.xml (hunk 1 above)
Comment (msingermann) [reply pom.xml]: done

**Pull request merged**